examples/mouse/main.go

```diff
@@ -4,21 +4,19 @@ package main
 // coordinates and events.
 
 import (
-	"fmt"
 	"log"
 
 	tea "github.com/charmbracelet/bubbletea"
 )
 
 func main() {
-	p := tea.NewProgram(model{}, tea.WithAltScreen(), tea.WithMouseAllMotion())
+	p := tea.NewProgram(model{}, tea.WithMouseAllMotion(), tea.WithMouseExtendedMode())
```

Comment:
Do we need both of these? Ideally, would be nice to get all the benefits of SGR mode with just one flag.

Reply:
We could have SGR mode enabled by default when using `WithMouseCellMotion` and `WithMouseAllMotion`, and then have `WithMousePixelsMode` to use SGR-Pixels.